README.md

```diff
@@ -178,20 +178,6 @@ To destroy the whole deployment:
 $ make destroy
 ```
 
-Alternatively, you can separately uninstall Kubernetes:
```

Comment:
I'd like to leave the option of uninstalling via `make uninstall` in here please. I quite often do `make uninstall` to cleanup the Kubernetes cluster before redeploying with `make install`

Reply:
What I mean is, I like just `destroy` as you've done it but could we keep the docs showing `make uninstall` too for those that want to uninstall Kubernetes but leave DC/OS cluster up.

Reply:
I see. I am thinking about a separate _Uninstalling Kubernetes_ section, to keep confusion to a minimum between `uninstall` and `destroy`.

Reply:
@jimmidyson https://github.com/mesosphere/dcos-kubernetes-quickstart/pull/69/files#diff-04c6e90faac2675aa89e2176d2eec7d8R173